tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/MockListener.java

```diff
@@ -61,4 +66,14 @@ public void notifyFailure(BError error) {
         }
         return err;
     }
+
+    public static BFuture invokeResourceWithUnionReturn(Environment env, BString name) {
+        if (service != null) {
```

Comment:
Do we need this null check?

Reply:
ATM it has been only used for a test which involve with a service test case. Hence the service will not be null here and we can remove it.